source/deployment/cluster.rst

```diff
@@ -431,6 +431,19 @@ For example, if you're deploying Mattermost on AWS with Amazon Aurora we recomme
 Troubleshooting
 ---------------
 
+Capturing High Availability Troubleshooting Data
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When deploying Mattermost in a High Availability configuration, we recommend that you keep Prometheus and Grafana metrics as well as cluster server logs for as long as possible - and at minimum two weeks. 
+
+You may be asked to provide this data to Mattermost for analysis and troubleshooting purposes.
+
+.. note::
+
+  - Ensure that server log files are being created. See :ref:`troubleshooting` for details.
```

Comment:
@cwarnermm Wondering what the ```:ref: `troubleshooting` ``` refers to here?

Reply:
@wiersgallak The intention was to provide a link to https://docs.mattermost.com/install/troubleshooting.html#review-mattermost-logs; however, I suspect that my relative link may not work as intended. @justinegeffen for this internal link, is the ```#review-mattermost-logs``` also required to resolve correctly?

Reply:
@wiersgallak & @justinegeffen - I've fixed this relative link by replacing it with a hypertext link for now.